app/src/main/java/co/develhope/chooseyouowncocktail_g2/ui/search/SearchFragment.kt

```diff
@@ -100,9 +97,25 @@ class SearchFragment : Fragment() {
                     )
                 }
             }
-            DrinkAction.SetPref -> TODO()
-        }}
+            is DrinkAction.SetPref -> {
+                DrinkList.setFavorite(action.drink, action.drinkPref)
+                DrinkList.booleanSortDrinkList()
+                DrinkList.drinkList().forEach{Log.d("debug", "${it.name} and ${it.favourite}") }
```

Comment:
Per debuggare occorrerebbe usare il debugger, ma in ogni caso non andrebbero inclusi in una PR.